### ISSUE TITLE
Errors bridging USDK (v1)

### ISSUE THREAD

**Issue by lucykite** [solana-labs/oyster#535]
I am trying to get my USDK (v1) bridged out. As instructed I went to https://v1.wormholebridge.com/#/move?from=SOL&token=USDK and hit transfer. The progress dialog got stuck on "Executing Solana Transaction" even though the transaction shows as success / finalized in Solscan. I went to interstellar.tk and copied in the tx-id, but the ETH transaction failed with "VAA version incompatible." This has happened with multiple test transactions; I am waiting for instructions before sending the full amount.

Solana tx: FreH6knvH6vxU6GzrkU4MhUgqCS3toFR2hGFdb9yAm1dM9XqCJtvFTtQK7CULBBqsY8KqFSMaWvS8dEFun8sTED
Ethereum tx: 0x937f70334767035cd29a2579b7e2ba8cd8844ff7c02db467c62969d317204af2

I am concerned that we need to resolve this ASAP before the bridge is deprecated at end-of-month.

**Comment by ckeun** [comment]
can you try redeeming it again via interstellar.tk?
i think you might have two txns to redeem?
5gXtjGyaCByy3Df2AgJ7J4dDCVg4MH7GJKFdTLspda5Qnr7MTyxcDZPF5zEX4dnPKo4tHxMHf9uiBFxfDiPgjqnF
FreH6knvH6vxU6GzrkU4MhUgqCS3toFR2hGFdb9yAm1dM9XqCJtvFTtQK7CULBBqsY8KqFSMaWvS8dEFun8sTED

both should be good now.

**Comment by lucykite** [comment]
Thanks - these were test transactions. I am waiting to transfer the full amount until I can be confident the process will succeed. Will future transactions work? Is there something I should do differently in order to have this succeed without intervention?

**Comment by ckeun** [comment]
I don't think you did anything wrong in terms of the transaction. There just weren't enough guardians up to sign the vaa. We appreciate you trying to migrate your stuff out of v1 👍 And i would recommend migrating your full amount over asap. it might take a day or two, but we'll help you make sure it all gets there.

**Comment by bdolin** [comment]
Done, thank you!

**Comment by lucykite** [comment]
Done, thanks!

**Comment by ckeun** [comment]
awesome, i'll keep eyes on it! 👀

**Comment by ckeun** [comment]
oh i think you successfully sent some more over! 🥳 congrats & thanks again for migrating
